lib/iris/cube.py

```diff
@@ -505,6 +510,16 @@ def concatenate_cube(self, check_aux_coords=True):
             is not applied to auxilliary coordinates that span the dimension
             the concatenation is occuring along. Defaults to True.
 
+        * check_cell_measures
+            Checks the cell measures of the cubes match. This check
+            is not applied to cell measures that span the dimension
+            the concatenation is occuring along. Defaults to True.
```

Comment:
```suggestion
            the concatenation is occurring along. Defaults to True.
```

These spellings are actually _easier_ as edit suggestions.
But you can't do it to a multi-line comment.